conf/messages.en

```diff
@@ -407,6 +407,19 @@ pensionCommencementLumpSum.title = Did any member receive a pension commencement
 pensionCommencementLumpSum.heading = Did any member receive a pension commencement lump sum (PCLS) during the period of this return?
 pensionCommencementLumpSum.error.required = Select yes if any member received a Pension Commencement Lump Sum during the period of this return
 
+pensionCommencementLumpSumAmount.title = Amount of PCLS and relevant pension for this member - Managing Pension Schemes - GOV.UK
```

Comment:
Should be `Amount of PCLS and relevant pension for this member`,
as `Managing Pension Schemes - GOV.UK - Managing Pension Schemes - GOV.UK` is already there by default